internal/pkg/install/install.go

```diff
@@ -120,9 +125,57 @@ func Install(data *userdata.UserData) (err error) {
 			}
 		}
 	}
+
+	if err = writeExtLinux(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func writeExtLinux() (err error) {
+	const extlinuxConfig = `DEFAULT Talos
+  SAY Talos ({{ .Version }}) by Autonomy
+LABEL Talos
+  KERNEL /boot/vmlinuz
+  INITRD /boot/initramfs.xz
+  APPEND page_poison=1 slab_nomerge pti=on nvme_core.io_timeout=4294967295 consoleblank=0 console=tty0 console=ttyS0,9600 talos.autonomy.io/userdata=http:/127.0.0.1/master.yaml talos.autonomy.io/platform=baremetal
+`
+	aux := struct {
+		Version string
+	}{
+		Version: version.Tag,
+	}
+	b := []byte{}
+	wr := bytes.NewBuffer(b)
+	t := template.Must(template.New("extlinux").Parse(extlinuxConfig))
+	if err = t.Execute(wr, aux); err != nil {
+		return err
+	}
+
+	if err = ioutil.WriteFile("/boot/extlinux/extlinux.conf", wr.Bytes(), 0600); err != nil {
```

Comment:
Do we need to `mkdir /boot/extlinux` ? That may be a gap in the current install bits.

Reply:
Yeah I think so.